core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -316,6 +332,46 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Sets the {@link ChannelOption} of the server socket bound by {@link Server}.
+     * Note that the previously added option will be overridden if the same option is set again.
+     *
+     * <pre>{@code
+     * ServerBuilder sb = new ServerBuilder();
+     * sb.channelOption(ChannelOption.BACKLOG, 1024)
+     *   .channelOption(ChannelOption.WRITE_BUFFER_WATER_MARK, 32 * 1024);
```

Comment:
I think `WRITE_BUFFER_WATER_MARK` is only for child channels. How about just removing the second line, given that the first line is good enough?